retrieval_service/example-config-cloudsql-mysql.yml

```diff
@@ -0,0 +1,11 @@
+host: 0.0.0.0
```

Comment:
this file can be removed. This is used in the integration test (cloudbuild.yaml file). And I used the same file for both cloudsql-pg and cloudsql-mysql. Eventually we will be merging all these example-config files. :)

Reply:
done